verification/cbmc/proofs/aws_byte_cursor_advance_nospec/aws_byte_cursor_advance_nospec_harness.c

```diff
@@ -36,7 +36,11 @@ void aws_byte_cursor_advance_nospec_harness() {
     } else {
         assert(rv.ptr == old.ptr);
         assert(rv.len == len);
-        assert(cursor.ptr == old.ptr + len);
+        if (old_cur.ptr != NULL) {
+            assert(cursor.ptr == old_cur.ptr + len);
```

Comment:
Please double check.

`old_cur` -> `old`

Reply:
yeah... Sorry, I did a search and replace, thought the name is the same across the files... Now, I have ran the all proofs locally and all of them passed